 error[E0308]: mismatched types
    --> src/dns/serde.rs:259:25
     |
 258 |		String::from_value(
     |		------------------ arguments to this function are incorrect
 259 | /	&rec.get_data_by_key(constants::columns::NAME)
 260 | |	.ok_or_else(|| must_have_col_err(constants::columns::NAME))?,
     | |___________________________________________________________________^ expected `Value`, found `&Value`

--

--- a/src/dns/serde.rs
+++ b/src/dns/serde.rs
@@ -256,7 +256,7 @@ impl Query {
 
                 let name = Name::from_utf8(
                     String::from_value(
-                        &rec.get_data_by_key(constants::columns::NAME)
+                        rec.get_data_by_key(constants::columns::NAME)
                             .ok_or_else(|| must_have_col_err(constants::columns::NAME))?,
                     )
                     .map_err(|err| LabeledError {
